cmd/egress/cmd.go

```diff
@@ -71,7 +76,16 @@ are set correctly before execution.
 				os.Exit(1)
 			}
 
-			out := cli.ValidateEgress(ctx, config.vpcSubnetID, config.cloudImageID, config.kmsKeyID, config.timeout)
+			// Set Up Proxy
+			// Get CACERT from env if exists
+			cacert := os.Getenv("CACERT")
```

Comment:
I think `cacert` should be passed with a flag similar to what was implemented in `network-validator.go`

Reply:
indeed, I'll be updating the readme as well